### PR TITLE
fix: Remove legacy HA interface stubs

### PR DESCRIPTION
**Pull request by btorresgil**
## Description

> Note: Merge with "Rebase and Merge", not squash.

Removes some HA interface stubs and assigns no 'name' to the HA interface class. This fixes 2 issues:

Issue 1: HA interfaces created with args, instead of kwargs, were not getting the correct IP address because pan-os-python expected the first arg to be a name.

Issue 2: #350 - HA interface stubs were created in the HighAvailability instance, but some newer models of VM-Series do not have these interfaces (eg. ha1-backup), so the stub caused an error.

## How Has This Been Tested?

Tested on VM-Series that supports ha1-backup and VM-Series that does not support ha1-backup. Works on both.

## Types of changes

- Bug fix (non-breaking change which fixes an issue)

## Checklist

- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes if appropriate.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by Joe-Zhou-Pub** [comment]
@shinmog : Can you please review and approve this PR? Many thanks.

**Comment by btorresgil** [comment]
I did additional testing on 7.1.0 firewall on multiple platforms due to concerns of affecting older devices. Also tested equality testing to ensure, but this passed as well. ha1 is the only stub required on modern PAN-OS versions.

All tests checked out so I'll merge this and release.

File: test.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+from panos.firewall import Firewall
+from panos.ha import HA1, HA2, HighAvailability
+from panos.network import EthernetInterface
+
+
+def main():
+    logging.basicConfig(level=0)
+
+    primary_fw = Firewall("10.8.202.107", "admin", "paloalto")
```
Comment (shinmog) [inline test.py:12]: Accidental commit (on this whole file)...?  If not, should be in the live tests directory or something..?

File: panos/ha.py [hunk 2]
```diff
@@ -558,10 +559,5 @@ def _setup(self):
 
         # stubs
         self._stubs.add_profile(
-            "0.0.0",
-            "interface/ha1",
-            "interface/ha1-backup",
-            "interface/ha2",
-            "interface/ha2-backup",
-            "interface/ha3",
+            "0.0.0", "interface/ha1",
```
[the remark this answers is not in the document]
Comment (btorresgil) [reply panos/ha.py:562]: Looks like this goes back pretty far, through all support PAN-OS versions, so decided not to version it.

File: test.py (hunk 1 above)
Comment (btorresgil) [reply test.py]: whoops! Thanks for catching!

**Comment by btorresgil** [comment]
Reviewed and discussed w/ @shinmog . ok to merge.

**Pull request merged**